includes/definitions/hwg-ste2.yaml

```diff
@@ -4,9 +4,37 @@ type: environment
 icon: hwg
 over:
     - { graph: device_temperature, text: Temperature }
-    - { graph: device_humidity, text: Humidity }
```

Comment:
Why was this overview graph removed?

Reply:
It's not a standard shipped sensor, only the temperature sensor is shipped by default. The humidity optional, so it's better to remove this graph as over.